### PR TITLE
[MERGE] hr_payroll: Improve general UX (back2basics)

### PR DESCRIPTION
**Pull request by tivisse**
Com: https://github.com/odoo/odoo/pull/72402/commits
Ent: https://github.com/odoo/odoo/pull/72402/commits
Upg: odoo/upgrade#2583

TaskID: 2558171

### CODE REVIEW

File: addons/web/static/src/legacy/scss/fields.scss [hunk 1]
```diff
@@ -542,6 +542,9 @@
         font-weight: 500;
         @include o-text-overflow;
         transition: none; // remove transition to prevent badges from flickering at reload
+        &.o_field_empty {
+            display: none !important;
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/scss/fields.scss:546]: is `!important` really necessary?

Comment (tivisse) [reply addons/web/static/src/legacy/scss/fields.scss:546]: It was prior to the owl / css refactoring. I'll test it again. 

But at the moment I wrote this, it was :)

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**